erpnext/public/js/utils.js

```diff
@@ -304,6 +304,43 @@ $.extend(erpnext.utils, {
 			}
 			frappe.ui.form.make_quick_entry(doctype, null, null, new_doc);
 		});
+	},
+
+	add_web_template: function(frm, cdt, cdn) {
+		let row = frm.selected_doc;
+		frappe.model.with_doc("Web Template", row.web_template).then((doc) => {
```

Comment:
This duplication is not right. There is an ongoing PR in frappe that deals with this duplication.

https://github.com/frappe/frappe/pull/11285/files#diff-a7f86b0d7ba77ad1dc83597f12ece957R53

We can use that approach once that is merged.